pkg/slo-controller/noderesource/plugins/midresource/plugin.go

```diff
@@ -219,3 +220,11 @@ func (p *Plugin) calculate(strategy *configuration.ColocationStrategy, node *cor
 		},
 	}
 }
+
+func getFreePhysicalResourceOnNode(node *corev1.Node, nodeMetrics *slov1alpha1.NodeMetric) *corev1.ResourceList {
+	// Capacity - nodeUsed
+	nodeCapacity := resutil.GetNodeCapacity(node)
+	nodeUsed := nodeMetrics.Status.NodeMetric.NodeUsage.ResourceList
```

Comment:
need to check if the NodeUsage is valid

Reply:
Fixed 
I proposed to aggressively set nodeUnUsed to 0 when it is invalid, which is safer